packages/battery_plus/pubspec.yaml

```diff
@@ -14,31 +14,30 @@ flutter:
       ios:
         pluginClass: FLTBatteryPlusPlugin
```

Comment:
Just curious - is this naming because we copied code from somewhere else?
(`BatteryPlusIosPlugin` would match the others)